cmd/swarmctl/service/list.go

```diff
@@ -31,6 +31,17 @@ var (
 				return err
 			}
 
+			nr, err := c.ListNodes(common.Context(cmd), &api.ListNodesRequest{})
+			if err != nil {
+				return err
+			}
+			readyNodes := make(map[string]struct{})
+			for _, n := range nr.Nodes {
+				if n.Status.State == api.NodeStatus_READY {
```

Comment:
Oops, I already created a new PR #1270 for this, would you mind switching to that one?

By the way, regarding updating this PR, is that a preferred way from reviewer's perspective? I am afraid using force push will make the review comments inconsistent with the code once the PR's commit is updated by the force push. Creating a separate PR is also not perfect because it will switch the review context and add more overhead for both reviewer and committer. Another way is appending another commit which fixes the review comments on top of the original commits of this one. I don't know which way is mostly favored.